typings/ResultTypes.ts

```diff
@@ -0,0 +1,11 @@
+export interface Stat {
```

Comment:
Change the folder name to interface and use the same convention from the backend, let;s use
main.interface.ts